content/realtime/channels/channel-parameters/overview.textile

```diff
@@ -111,18 +111,29 @@ TBD
 TBD
 ```
 
-h2(#using-parameters-non-ably). Using channel parameters without Ably libraries
+h2(#using-parameters-non-ably). Using channel parameters outside of supported Ably libraries
 
-It is possible to interact with Ably channels using transports that do not involve using an Ably library; for example using SSE without any library, or using a supported non-Ably protocol such as MQTT. In these cases, it is also necessary to use a qualified channel name.
+For the client libraries that do not currently expose the API, or transports that do not involve using an Ably library, a set of channel paramaters can be expressed by including a query string with standard URL query syntax and encoding, within the qualifier part of a channel name. The qualifier part is in square brackets at the start of the channel name.
```

Comment:
For the client libraries that do not currently expose the **channel parameters** API

typo paramaters -> parameters